worker/machiner/machiner.go

```diff
@@ -169,7 +169,7 @@ func (mr *Machiner) Handle(_ <-chan struct{}) error {
 				return errors.Annotate(err, "cannot update observed network config")
 			}
 		}
-		logger.Debugf("observed network config updated for %q to %v", mr.config.Tag, observedConfig)
+		logger.Debugf("observed network config updated for %q to %+v", mr.config.Tag, observedConfig)
```

Comment:
I believe we already depend on kr/pretty, so you could do:
%# v", ... pretty.Formatter(observedConfig))
which gives you nicely delimited structs.
Consider it, see if you like it more/less.